classes/checkout/CheckoutStepInterface.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+interface CheckoutStepInterface
+{
+    public function render(array $extraParams = []);
+    public function getTitle();
+    public function handleRequest(array $requestParameters = []);
+    public function setCheckoutProcess(CheckoutProcess $checkoutProcess);
+    public function getCheckoutProcess();
```

Comment:
can't have protected interface methods, but removed the method since it actually belongs to the implementation